pkg/odo/cli/pipelines/wizard.go

```diff
@@ -90,6 +123,59 @@ func (io *WizardParameters) Complete(name string, cmd *cobra.Command, args []str
 	return nil
 }
 
+func checkBootstrapDependencies(io *WizardParameters, kubeClient kubernetes.Interface, spinner status) error {
+
+	var errs []error
+	client := utility.NewClient(kubeClient)
+	log.Progressf("\nChecking dependencies\n")
+
+	spinner.Start("Checking if Sealed Secrets is installed at kube-system namespace", false)
+	err := client.CheckIfSealedSecretsExists(types.NamespacedName{Namespace: sealedSecretsNS, Name: sealedSecretsName})
+	checkSpinner(spinner, "Please install Sealed Secrets from https://github.com/bitnami-labs/sealed-secrets/releases", err)
+	if err == nil {
+		io.SealedSecretsService.Name = sealedSecretsName
+		io.SealedSecretsService.Namespace = sealedSecretsNS
+	} else if !errors.IsNotFound(err) {
+		return clusterErr(err.Error())
+	}
+
+	spinner.Start("Checking if ArgoCD Operator is installed at argocd namespace", false)
+	err = client.CheckIfArgoCDExists(argoCDNS)
+	checkSpinner(spinner, "Please install ArgoCD operator from OperatorHub", err)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			return clusterErr(err.Error())
+		}
+		errs = append(errs, err)
+	}
+
+	spinner.Start("Checking if OpenShift Pipelines Operator is installed", false)
+	err = client.CheckIfPipelinesExists(pipelinesOperatorNS)
+	checkSpinner(spinner, "Please install OpenShift Pipelines operator from OperatorHub", err)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			return clusterErr(err.Error())
+		}
+		errs = append(errs, err)
+	}
+
+	if len(errs) > 0 {
+		return fmt.Errorf("Failed to satisfy the required dependencies")
+	}
+	return nil
+}
+
+func checkSpinner(spinner status, warningMsg string, err error) {
```

Comment:
`checkSpinner` doesn't really seem to describe what this does?

Reply:
Changed the name to `setSpinnerStatus`